src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/Interop.cryptoapi.cs

```diff
@@ -3,20 +3,24 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Text;
 using System.Diagnostics;
 using System.Runtime.InteropServices;
 
+
+using Internal.Cryptography;
 using Internal.Cryptography.Pal.Native;
 
 internal static partial class Interop
 {
-    public static class advapi32
+    public static partial class cryptoapi
     {
 #if !NETNATIVE
-        [DllImport(Libraries.Advapi32, SetLastError = true, CharSet = CharSet.Unicode, EntryPoint = "CryptAcquireContextW")]
+        [DllImport(Libraries.SecurityCryptoApi, SetLastError = true, CharSet = CharSet.Unicode, EntryPoint = "CryptAcquireContextW")]
```

Comment:
If this is using the new contract DLL I'd like to hold off until Win7 OuterLoop runs are succeeding; I'd like to be assured that we haven't broken our minimum OS by not having an up-to-date apiset package.